django/test/runner.py

```diff
@@ -448,6 +449,8 @@ def __init__(self, pattern=None, top_level=None, verbosity=1,
         self.parallel = parallel
         self.tags = set(tags or [])
         self.exclude_tags = set(exclude_tags or [])
+        if not no_faulthandler:
+            faulthandler.enable()
```

Comment:
You need to enable faulthandler in the `ParallelTestSuite`'s processes as well, as per comment on ticket.

Reply:
The `faulthandler.enable()` installs signal handlers which are inherited by the child processes being forked by `multiprocessing`, so the traceback is printed even when tests are being run in parallel and the child processes aborts.

Reply:
Ah right. You noted on the ticket that's not the case with the 'spawn' mode, but I guess we shouldn't concern ourselves with that right now, as the test runner doesn't support that.

@Valze - maybe make a note for your spawn-mode GSoC project? 😉

Reply:
Adding support for the 'spawn' mode might be pretty useful as macOS has switched to 'spawn' as the default starting at Python 3.8

Reply:
@adamchainz Sorry for the late reply, didn't catch this in my emails. 

Definitely noting this and adding it to my forum post. I'm taking my time going through my original PR and trying to flesh out the PR more and noting the current blockers.

Thanks for the catch.